realsense2_camera/src/rs_node_setup.cpp

```diff
@@ -327,7 +327,7 @@ void BaseRealSenseNode::startRGBDPublisherIfNeeded()
 }
```

Comment:
@hyunseok-yang ,  Please update the same for "rgbd" topic as well. Refer line number 319.

Reply:
Oh, I missed it!